src/js/components/Button/doc.js

```diff
@@ -46,7 +46,7 @@ export const doc = Button => {
     color: colorPropType.description(
       'Fill color for primary, label color for plain, border color otherwise.',
     ),
-    disabled: PropTypes.bool
+    disabled: PropTypes.oneOfType([PropTypes.bool, PropTypes.number])
```

Comment:
I don't think we want to add the 'number' type to button as discussed on previous threads.

Reply:
I know Im only doing this for testing purposes

Reply:
It wont allow me to add disabled test without it (if only we had it before we would of saw this error)